core/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ## [Unreleased] - ReleaseDate
 
+### Added
+
+- Added `Rgb666` type support.
```

Comment:
```suggestion
- [#621](https://github.com/embedded-graphics/embedded-graphics/pull/621) Added `Rgb666` color type support.
```

Please also add the same line to the top-level `CHANGELOG.md` too - these types are re-exported by e-g itself so it's good to let direct users know.